### PR TITLE
Verify and Test attestation tests (verified on ee64b4a)

### PR DESCRIPTION
**Pull request by anjalirai-intel**
[curated_app.log](https://github.com/bprashan/curated_apps/files/9443161/curated_app.log)
[curated_app_bash_runtime_args.log](https://github.com/bprashan/curated_apps/files/9443163/curated_app_bash_runtime_args.log)
[curated_app_invalid_encryption_key.log](https://github.com/bprashan/curated_apps/files/9443164/curated_app_invalid_encryption_key.log)

### CODE REVIEW

File: libs/config_parser.py [hunk 1]
```diff
@@ -66,3 +67,28 @@ def data_pre_processing_for_verifier_image(test_config_dict, end_test_key_str):
             shutil.copytree(test_config_dict["ssl_path"], VERIFIER_SERVICE_PATH + "/ssl")
     if "bash/bash" in test_config_dict['docker_image']:
         shutil.copytree("data/bash", CURATED_APPS_PATH + "/bash")
+        if utils.check_machine() == "Azure Linux Agent":
+            bash_azure_setup()
+
+def bash_azure_setup():
+    bash_dir = os.path.join(CURATED_APPS_PATH, "bash")
```
Comment (bprashan) [inline libs/config_parser.py:74]: Can we use BASH_PATH variable from constants.py?

File: libs/config_parser.py [hunk 2]
```diff
@@ -66,3 +67,28 @@ def data_pre_processing_for_verifier_image(test_config_dict, end_test_key_str):
...
+
+    # Updating dockerfile for bash
+    fd = open(os.path.join(bash_dir, "Dockerfile"))
+    fd_contents = fd.read()
+    fd.close()
+    new_data = re.sub('ENV http_proxy "http://proxy-dmz.intel.com:911"\nENV https_proxy "http://proxy-dmz.intel.com:912"\n',
```
Comment (bprashan) [inline libs/config_parser.py:80]: **re.sub** function is used in conftest.py as well. Can we have a common function in utils.py?

Comment (anjalirai-intel) [reply libs/config_parser.py]: Done

**Comment by anjalirai-intel** [comment]
Verified on commit: [ee64b4a5d6b2d0cba62e18b5c6c36b070bca56c0](https://github.com/aneessahib/contrib/commit/ee64b4a5d6b2d0cba62e18b5c6c36b070bca56c0)

**Review by bprashan — APPROVED** [review]

**Pull request merged**